ql/termstructures/credit/defaultprobabilityhelpers.cpp

```diff
@@ -257,7 +257,6 @@ namespace QuantLib {
     }
 
     void UpfrontCdsHelper::initializeDates() {
-        CdsHelper::initializeDates();
```

Comment:
Why is this no longer called?

Reply:
`UpfrontCdsHelper::initializeDates()` is private and only called in the `UpfrontCdsHelper` constructor but `CdsHelper::initializeDates()` has already been called in the base class constructor call so you end up calling it twice.